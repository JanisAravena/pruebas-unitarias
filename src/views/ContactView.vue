<template>
    <div>
        <h1>Vista de Contacto</h1>
    </div>
</template>

<script>
export default {
    name: 'ContactViewVue'
}
</script>

<style scoped></style>