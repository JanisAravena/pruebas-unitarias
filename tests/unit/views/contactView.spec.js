import { shallowMount } from "@vue/test-utils";
import ContactView from "@/views/ContactView.vue";

describe("ContactView", () => {
  test("Coincide con el snapshot", () => {
    const wrapper = shallowMount(ContactView);
    expect(wrapper.element).toMatchSnapshot();
  });
});