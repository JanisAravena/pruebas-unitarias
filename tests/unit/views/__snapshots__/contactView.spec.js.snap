// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactView Coincide con el snapshot 1`] = `
<div>
  <h1>
    Vista de Contacto
  </h1>
</div>
`;
